resources/testdata/glyphs2/WghtVar_ImplicitAxes.glyphs

```diff
@@ -1,9 +1,5 @@
 {
-.appVersion = "3148";
-DisplayStrings = (
-"-",
-"!"
-);
+.appVersion = "3340";
```

Comment:
doesn't this mean glyphs3?

Reply:
no, that's simply saying this particular glyphs2 source file was serialized by Glyphs 3 (3340). you can 'save as'. I don't have a Glyphs 2.app any more.
The .formatVersion key stores the source format version, it's omitted in Glyphs 2 (defaults to 2 in fact).